preparing-to-code/ada-languages/README.md

```diff
@@ -30,11 +30,24 @@ In Addition, these compiled and interpreted languages can be divided into dynami
 
 ## Languages
 
-### Ruby
+### Python
 
-[Ruby](https://www.ruby-lang.org/en/) is a high-level, dynamically typed language.  Ruby is different than other programming languages. It was written with human communication in mind, rather than computer optimization. Its creator wanted to create a programming language that was fun for programmers. Ruby is a programming language which was created to make developers have fun and be productive at the same time. Ruby was first written in 1993 by [Yukihiro "Matz" Matsumoto](https://twitter.com/yukihiro_matz). _"Ruby is for People. I hope to see Ruby help every programmer in the world to be productive, and to enjoy programming, and to be happy. That is the primary purpose of Ruby language."_ - Matz
+[Python](https://www.python.org) is a high-level, dynamically typed language.  It was created by [Guido van Rossum](https://en.wikipedia.org/wiki/Guido_van_Rossum) in 1991.  Originally Python was a hobby programming project to Van Rossum wrote to keep busy during the holidays.  He chose the name Python because he is a fan of [Monty Python's Flying Circus](https://en.wikipedia.org/wiki/Monty_Python%27s_Flying_Circus).
```

Comment:
I haven't heard about the project to keep busy during the Holidays.  Might be a fun link since this paragraph is a little light.

Reply:
It's in the wikipedia article.